app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/layout/default.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="head.components">
```

Comment:
// need to remove all unnecessary

<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>
            <move element="form.subscribe" destination="main.content" as="new_form.subscribe" after="-" before="-"/>
    </body>
</page>

Reply:
Done